pkg/plugins/runtime/k8s/controllers/service_controller.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"fmt"
 
-	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
in other parts of the code it's called `apiextensionsv1`, do you think we could make it consistent?